frontend/src/utils.ts

```diff
@@ -30,18 +31,24 @@ export const routeToPage = (mediaId: string, mediaType: string = undefined) => {
 
             switch (startingChar) {
                 case "t":
-                    window.location.href = `/tv/${slicedId}`;
+                    url = `/tv/${slicedId}`;
                     break;
                 case "m":
-                    window.location.href = `/movie/${slicedId}`;
+                    url = `/movie/${slicedId}`;
                     break;
                 case "p":
-                    window.location.href = `/person/${slicedId}`;
+                    url = `/person/${slicedId}`;
                     break;
                 default:
-                    window.location.href = "/";
+                    url = "/";
             }
     }
+
+    if (ctrlDown) {
+        window.open(url);
+    } else {
+        window.location.href = url;
+    }
```

Comment:
Do you know if there is a native JS/TS way to get this behavior where we wont have to describe every type of click (that also supports middle mouse + shift click)?

Reply:
You *should* be able to check which mouse button is pressed by checking the button state 
[https://developer.mozilla.org/en-US/docs/Web/API/Pointer_events#determining_button_states](https://developer.mozilla.org/en-US/docs/Web/API/Pointer_events#determining_button_states)

The PointerEvent where the ctrlDown boolean comes from also support modifier keys like alt and shift. 

If you are interested in non modifier keys I believe you have to do something like this. While has been said I don't really have that much experience with Svelte, so there might be an easier way to do it. 

```ts
let lastKeyPress: string | null = null;

window.addEventListener("keydown", (e: KeyboardEvent) => {
  lastKeyPress = e?.key;
});

window.addEventListener("keyup", (e: KeyboardEvent) => {
  lastKeyPress = null;
});

```